pkgs/data/icons/nordzy-icon-theme/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+, gtk3
```

Comment:
```suggestion
, gtk3
, jdupes
```

Reply:
Use `jdupes` to replace duplicate files with hard links, considerable reducing the needed disk space. See below.